haven't gottne to check whether i made mistakes yet, but i added chec…

pinoaffe opened the pull request:
haven't gotten to check whether i made mistakes yet, 
but i added checks for the existence of a user to a few database interaction functions and added some comments
todo:
modify the functions to throw exceptions and add email to check_user_exists. Maybe also add an check_article_exists function.

scheurneus commented:
Looks good. Some small points:

- The comment spacing is inconsistent, and maybe split very long comment lines in multiple?
- Perhaps add a check_user_exists to push_article.

Also, have you tested the workings?

pinoaffe commented:
i've made the comments more consistent and split long comments, also added a user-existence check to push_article.
I have yet to do tests, couldn't get the mysql module to work on my laptop for some reason
